app/controllers/products_controller.rb

```diff
@@ -1,40 +1,50 @@
 class ProductsController < ApplicationController
-  before_action :set_product, only: [:show, :edit, :update]
-  
+before_action :set_product, only: [:show, :edit, :update]
+
+  def show
+    @booking = Booking.new
+    @product = Product.find(params[:id])
+  end
+
+  def index
+     @products = Product.all
+  end
+
+
   def new
     @product = Product.new
   end
-    
-  def create 
+
+  def create
     @product = Product.new(product_params)
     @product.user = current_user
     if @product.save
       redirect_to product_path(@product)
     else
       render :new
     end
-  end  
-    
+  end
+
 
   def edit
   end
-  
+
   def update
     if @product.update(product_params)
-      redirect_to product_path(@product) 
+      redirect_to product_path(@product)
     else
       render :edit
     end
   end
-    
-    
-    private 
+
+
+    private
 
     def product_params
-      params.require(:product).permit(:title, :description, :status, :condition, :address)  
+      params.require(:product).permit(:title, :description, :status, :condition, :address)
```

Comment:
Similiar with the bookings status, it will get a default value that we have to decide on and that the user does not have to handle